### DISCUSSION TITLE
How Use subdomain in vercel domains

### DISCUSSION THREAD

**Discussion by moochieng** [vercel/community#1332] [Help] [answered]
### Summary

I bought this domain `efacke.org` domain and I want to host one  my  vercel project with a domain like this `admin.efacke.org` and another `community.efacke.org` how do I go about this

**Comment by domainsbasics** [comment, marked as answer]
if you haven't already, can you try the steps here
https://vercel.com/guides/how-do-i-add-a-custom-domain-to-my-vercel-project and add the subdomains

**Reply by amyegan** [reply]
These docs and guides might also be useful:
- [Domains Overview: Subdomains](https://vercel.com/docs/concepts/projects/domains/add-a-domain#subdomains)
- [Can I redirect from a subdomain to a subpath?](https://vercel.com/guides/can-i-redirect-from-a-subdomain-to-a-subpath)
- [How can I serve multiple projects under a single domain?](https://vercel.com/guides/how-can-i-serve-multiple-projects-under-a-single-domain#deploying-rewrite-rules)
- [How do I point a subdomain to a service outside of Vercel?](https://vercel.com/guides/pointing-subdomains-to-external-services)
